site/docs/getting-started/v3/microsoft-testing-platform.md

```diff
@@ -235,7 +232,7 @@ Actual:   False
 Failed!  - Failed:     2, Passed:     3, Skipped:     0, Total:     5, Duration: 33 ms - MyFirstUnitTests.dll (net8.0)
 ```
 
-To enable the new `dotnet test` experience, add the following property to your project file (.csproj/.fsproj/.vbproj):
+To enable MTP compatibility with the VSTest mode of `dotnet test`, add the following property to your project file (.csproj/.fsproj/.vbproj):
```

Comment:
I'm making this change to not confuse this with the new `dotnet test` experience of .NET 10 SDK.